src/apiCalls.tsx

```diff
@@ -14,7 +14,7 @@ export const getAllCocktails = async (): Promise<AllCocktailsDetails> => {
   return data;
 };
 
-export const getRandomCocktail = async ():Promise< { [key: string]: string }> => {
+export const getRandomCocktail = async ():Promise<RandomCocktail> => {
```

Comment:
I saw this in apiCalls and was wondering what the deleted file was doing.  But it appears that it needs to be interface for the expected object. Nice.

Reply:
It technically bypasses the strictness of TS (so isn't recommended) and was a fix that Becca had found to get the removeNulls functionality working but that's been put on the backburner and Hannah helped us get a solution for it that was following TS's rules.

Reply:
I get it - saying that there will be an object returned that has a key/value that really can be anything!  Thanks Alex.